cms/static/cms/js/modules/nextuntil.js

```diff
@@ -0,0 +1,24 @@
+var $ = require('jquery');
```

Comment:
jquery.nextuntil won't be linted :)
don't think it's really necessary to indicate the dependency in the filename, now that it's explicit in the source